README.md

```diff
@@ -9,15 +9,14 @@ Integration Partners
 
 Agnostic Cloud Provider
 
-[![Provider Agnostic Regression Linux](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/OpenEnclave/job/Agnostic-Linux-Badge/badge/icon?subject=Provider%20Agnostic%20Regession)](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/OpenEnclave/job/Agnostic-Linux-Badge/)
+[![Provider Agnostic Regression Linux](https://openenclaveci.westus.cloudapp.azure.com/job/OpenEnclave/job/Badges/job/Agnostic-Linux/badge/icon)](https://openenclaveci.westus.cloudapp.azure.com/job/OpenEnclave/job/Badges/job/Agnostic-Linux/)
 
 Azure
 
-[![Nightly Regression](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/OpenEnclave/job/Nightly/badge/icon?subject=Azure%20Regression%20Testing)](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/OpenEnclave/job/Nightly/)
```

Comment:
Note that I completely removed the badge for nightly regression because it is just the combination of "Provider Agnostic Regression Linux", "Azure Windows", and "Azure Linux". In my opinion the same results do not need to be reported twice. Please let me know if there are any opinions on this.